app/views/mfa_confirmation/show.html.erb

```diff
@@ -1,4 +1,4 @@
-<% self.title = t('titles.mfa_setup.suggest_second_mfa') %>
+<% self.title = @content.heading %>
```

Comment:
I'd consider updating the existing spec here, since it seems to be hard-coded to assume the `suggest_second_mfa` string will be assigned as the title, but it's now a little more dynamic than that.

https://github.com/18F/identity-idp/blob/e963df5c33754e232e9221980e014f7424d3d497/spec/views/mfa_confirmation/show.html.erb_spec.rb#L12-L16

I think we could rewrite the test to assert that the title is assigned to whatever the result of the presenter's `heading` method is.

Maybe something like (untested):

```rb
  it 'has a localized title' do
    expect(presenter).to receive(:heading).and_return('example')
    expect(view).to receive(:title=).with('example')

    render
  end
```

Reply:
I can do that. The bottom of the file has a suite set up for face/touch unlock setup, so I can add the test there.